frontend/alert/src/components/AutoComplete.js

```diff
@@ -206,14 +241,29 @@ const AutoComplete = () => {
                 autocomplete="off"
                 placeholder="Course"
                 ref={setInputRef}
-                value={value}
-                onChange={({ target: { value: newValue } }) => {
-                    setValue(newValue);
-                }}
-                onKeyDown={(e) => {
-                    // autocomplete with backdrop when the right arrow key is pressed
+                onKeyUp={(e) => {
                     if (e.keyCode === 39 && inputRef && suggestions && suggestions[0]) {
```

Comment:
```suggestion
                    if (e.keyCode === RIGHT_ARROW && inputRef && suggestions && suggestions[0]) {
```

Magic numbers for `keyCode`s should be defined as constants